config/config.yaml

```diff
@@ -1,22 +1,22 @@
-serverPort: 8080
+serverPort: ${KALDB_SERVER_PORT:-8080}
 
 indexerConfig:
-  maxMessagesPerChunk: 100
-  maxBytesPerChunk: 100000
-  commitDurationSecs: 10
-  refreshDurationSecs: 11
-  staleDurationSecs: 7200
+  maxMessagesPerChunk: ${INDEXER_MAX_MESSAGES_PER_CHUNK:-100}
```

Comment:
I think it's best to bump up this default to 500k. Otherwise, we may have chunk roll over exceptions?